boolean/boolean.py

```diff
@@ -1073,7 +1080,7 @@ def pretty(self, indent=0, debug=False):
         For example::
 
             >>> print(BooleanAlgebra().parse(
-            ...    u'not a and not b and not (a and ba and c) and c or c').pretty())
```

Comment:
> why would a >>> prefix be needed? that still would not be a proper interpreter session then?

because in rst, `>>>` and `...` start code blocks, and sphinx will print a warning in deep red about it :)
This should be put in an equivalent of markdowns
````
```
>>> bla
... bla
... bla
```
````

Tbh I'm not a rst expert, so if you have any clues how to adjust it, feel free to suggest a change.

Reply:
Thanks... let me check this by building the doc locally and may be push an update in your branch if I can find a solution.